xwiki-platform-core/xwiki-platform-security/xwiki-platform-security-authentication/xwiki-platform-security-authentication-script/src/main/java/org/xwiki/security/authentication/script/AuthenticationScriptService.java

```diff
@@ -240,4 +245,21 @@ public void resetPassword(UserReference user, String verificationCode, String ne
         this.resetPasswordManager.checkVerificationCode(user, verificationCode);
         this.resetPasswordManager.resetPassword(user, newPassword);
     }
+
+    /**
+     * Retrieve users information associated to the given email address and send them by email.
+     *
+     * @param userEmail the email address for which to find associated accounts
+     * @throws RetrieveUsernameException in case of problem for finding the information or for sending the email
+     * @since 14.9
+     * @since 13.10.10
+     * @since 14.4.6
+     */
+    public void retrieveUsernameAndSendEmail(String userEmail) throws RetrieveUsernameException
```

Comment:
Should this be marked with `@Unstable` ?

Reply:
Right, I missed that one.